overlay.d/99okd/etc/sysusers.d/openvswitch.conf

```diff
@@ -0,0 +1,4 @@
+#Type Name           ID             GECOS                   Home directory Shell
+u     openvswitch    984:976        "Open vSwitch Daemons"  /              /sbin/nologin
+g     openvswitch    976
+m     openvswitch    hugetlbfs
```

Comment:
Do you think we might be able to convince the RPM maintainers to install this file instead of running {group,user}add in %pre?

Reply:
We should encourage them to use `sysusers`, that'd help in the future

Reply:
Agreed, I'll file a PR on the rpm spec then

Reply:
That was actually backported to F36. We should be able to drop it here now.